### PR TITLE
Split work

### PR DESCRIPTION
**Pull request by motajessica**
As we spoke, this is just to make it easier for us to work in different features without creating conflicts by working on the same files.

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    // "jsxFragmentFactory": "",                         /* Specify the JSX Fragment reference used for fragments when targeting React JSX emit e.g. 'React.Fragment' or 'Fragment'. */
+    // "jsxImportSource": "",                            /* Specify module specifier used to import the JSX factory functions when using 'jsx: react-jsx*'. */
+    // "reactNamespace": "",                             /* Specify the object invoked for 'createElement'. This only applies when targeting 'react' JSX emit. */
+    // "noLib": true,                                    /* Disable including any library files, including the default lib.d.ts. */
+    // "useDefineForClassFields": true,                  /* Emit ECMAScript-standard-compliant class fields. */
+    // "moduleDetection": "auto",                        /* Control what method is used to detect module-format JS files. */
+
+    /* Modules */
+    "module": "commonjs",                                /* Specify what module code is generated. */
+    // "rootDir": "./",                                  /* Specify the root folder within your source files. */
```
Comment (Rainbow2544) [inline tsconfig.json]: Please change to “rootDir” : “./src”

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    // "maxNodeModuleJsDepth": 1,                        /* Specify the maximum folder depth used for checking JavaScript files from 'node_modules'. Only applicable with 'allowJs'. */
+
+    /* Emit */
+    // "declaration": true,                              /* Generate .d.ts files from TypeScript and JavaScript files in your project. */
+    // "declarationMap": true,                           /* Create sourcemaps for d.ts files. */
+    // "emitDeclarationOnly": true,                      /* Only output d.ts files and not JavaScript files. */
+    // "sourceMap": true,                                /* Create source map files for emitted JavaScript files. */
+    // "inlineSourceMap": true,                          /* Include sourcemap files inside the emitted JavaScript. */
+    // "outFile": "./",                                  /* Specify a file that bundles all outputs into one JavaScript file. If 'declaration' is true, also designates a file that bundles all .d.ts output. */
+    // "outDir": "./",                                   /* Specify an output folder for all emitted files. */
```
Comment (Rainbow2544) [inline tsconfig.json]: please change to “outDir” : ”./build”

File: package.json [hunk 3]
```diff
@@ -5,10 +5,9 @@
 	"main": "server.js",
 	"type": "module",
 	"scripts": {
-		"start": "nodemon src/server.js",
-		"test": "echo \"Error: no test specified\" && exit 1"
+		"start": "nodemon src/server.js"
```
Comment (Rainbow2544) [inline package.json]: Do we better add "build": "tsc" and "test": "jest" here, so we do not need to merge later?

File: package.json [hunk 4]
```diff
@@ -5,10 +5,9 @@
...
 	},
-	"author": "Ram Manthry",
+	"author": "",
 	"license": "ISC",
 	"dependencies": {
 		"dotenv": "^16.0.3",
```
Comment (Rainbow2544) [inline package.json:15]: Do we better install the typescript and Jest, and set up the jest.config.js in this base api first?

File: package.json (hunk 3 above)
Comment (motajessica) [reply package.json]: I have made the changes here

File: package.json (hunk 4 above)
Comment (motajessica) [reply package.json:15]: Done

File: tsconfig.json (hunk 2 above)
Comment (motajessica) [reply tsconfig.json]: done

**Review by Rainbow2544 — APPROVED** [review]

**Pull request merged**